views/help/contact-card.hbs

```diff
@@ -16,12 +16,12 @@
     <form class="wish_form hidden" action={{concat "/helpdesk?_csrf=" @root.csrfToken}} method="post" enctype="multipart/form-data">
       {{> "help/forms/form_wish"}}
       {{> "lib/components/csrfInput"}}
-      <button type="submit">{{$t "global.button.submit" }}</button>
+      <button type="submit" id="wish_submit">{{$t "global.button.submit" }}</button>
```

Comment:
You use different casing for ids and classes in your changes. I just looked for best practices and mostly it is adviced to use kebab-case, like here e.g.: https://makersaid.com/how-to-name-html-ids-and-classes/

Reply:
Chosen package manipulates the class names from kebap to snake case, so things get even more messy. Therefore i will leave everything as it is. Vue can be the only answer here ;-)